### ISSUE TITLE
training log

### ISSUE THREAD

**Issue by snsun** [sthalles/SimCLR#41]
Hi, 
Do you still have the log of pre-training? I want to know how the loss changes for every epoch and the accuracy of the positive examples in each batch

**Comment by HenryPengZou** [comment]
I have the same question here
